rcloud.support/R/rcloud.support.R

```diff
@@ -495,8 +495,14 @@ rcloud.is.notebook.visible <- function(id)
     visibility
   }
 
-rcloud.set.notebook.visibility <- function(id, value)
+rcloud.set.notebook.visibility <- function(id, value){
   rcloud.set.notebook.property(id, "visible", value != 0);
+  if(value){
+    response <- update.solr(rcloud.get.notebook(id),rcloud.notebook.star.count(id))
```

Comment:
Looks like what we are doing in other cases is indexing the base64-encoded gunk, haha. So I guess by that standard, we can just fetch with raw=TRUE. But that's pretty terrible...